docs/src/content/docs/ja/getting-started.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: 入門
+description: AstroのStarlightで新しいドキュメントサイトを作成する方法について学びます。
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+:::caution[作業中]
+Starlightはまだ開発の初期段階にあり、開発が進むにつれバグや変更が生じることが予想されます。何か不具合を見つけた場合は、[GitHubでissueを開く](https://github.com/withastro/starlight/issues/new/choose)か、または[Discord](https://astro.build/chat)でお知らせください。
+:::
+
+## 新しいプロジェクトを作成する
+
+Starlightは、オールインワンのフレームワークである[Astro](https://astro.build)の上に構築されています。次のコマンドにより、Astro + Starlightの新しいプロジェクトを作成できます。
+
+<Tabs>
+<TabItem label="npm">
+
+```sh
+# npmで新しいプロジェクトを作成する
+npm create astro@latest -- --template starlight
+```
+
+</TabItem>
+<TabItem label="pnpm">
+
+```sh
+# pnpmで新しいプロジェクトを作成する
+pnpm create astro --template starlight
+```
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+# yarnで新しいプロジェクトを作成する
+yarn create astro --template starlight
+```
+
+</TabItem>
+</Tabs>
+
+これにより、サイトに必要なすべてのファイルと設定が含まれた、新しいプロジェクトディレクトリが作成されます。
+
```

Comment:
In 19a65c57a7eab42bbce3904fc44196fc1c4c20bf I added an extra `:::tip` here if you’d like to add that before we merge this.